keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes_test.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import numpy as np
+import tensorflow as tf
+
+from keras_cv.layers.preprocessing3d import base_augmentation_layer_3d
+from keras_cv.layers.preprocessing3d.group_points_by_bounding_boxes import (
+    GroupPointsByBoundingBoxes,
+)
+
+POINT_CLOUDS = base_augmentation_layer_3d.POINT_CLOUDS
+BOUNDING_BOXES = base_augmentation_layer_3d.BOUNDING_BOXES
+OBJECT_POINT_CLOUDS = base_augmentation_layer_3d.OBJECT_POINT_CLOUDS
+OBJECT_BOUNDING_BOXES = base_augmentation_layer_3d.OBJECT_BOUNDING_BOXES
+
+
+class GroupPointsByBoundingBoxesTest(tf.test.TestCase):
+    def test_augment_point_clouds_and_bounding_boxes(self):
```

Comment:
It's probably worth including a test case which demonstrates that:

- calling the layer with training=False is a noop
- the layer preserves the input point clouds and bounding boxes (and potentially any other keys thrown at it)

Reply:
1. Done.
2. Do you mean the following?
```
        self.assertAllClose(inputs[POINT_CLOUDS], outputs[POINT_CLOUDS])
        self.assertAllClose(inputs[BOUNDING_BOXES], outputs[BOUNDING_BOXES])
```

Reply:
Yes I think that would be reasonable (and maybe also passing some other random key as an input to the layer to make sure it gets preserved in the output)